ext/exif/tests/exif028.phpt

```diff
@@ -0,0 +1,145 @@
+--TEST--
+Check for exif_read_data, HEIF with IFD0 and EXIF data in Motorola byte-order.
+--EXTENSIONS--
+exif
+--INI--
+output_handler=
+zlib.output_compression=0
+--FILE--
+<?php
+var_dump(exif_read_data(__DIR__.'/../../standard/tests/image/test4pix.heic'));
```

Comment:
exif should use it's own image as it's a separate extension. There are currently no images shared between exif and ext standard so it would be better to keep that way.

Reply:
the image file is 42199 bytes, are you sure we need add it 2 times? BTW, iphone will generate fake HEIF file if the image size is too small.